<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>691</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="myPainterWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>798</width>
          <height>498</height>
         </rect>
        </property>
        <widget class="QLabel" name="currentFlag">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>160</y>
           <width>60</width>
           <height>60</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/images/img/flag.png</pixmap>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="idLabel">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>ID：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commitbyLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>CommitBy：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="committimeLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>CommitTime：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="commitTime">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commentLabel">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="text">
            <string>Comment：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="commentTextEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="switchToNodeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>切换工作区到此节点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pullFromCommitButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>从此节点拉取/合并</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="diffWithNodeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>对比此节点与工作区</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="workSpaceWidget">
       <item>
        <property name="text">
         <string>+ /asdasdasdaf</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="Dense2Pattern">
          <color alpha="255">
           <red>189</red>
           <green>250</green>
           <blue>186</blue>
          </color>
         </brush>
        </property>
        <property name="foreground">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-  /gkuhhvkhbu</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="Dense6Pattern">
          <color alpha="255">
           <red>255</red>
           <green>138</green>
           <blue>138</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*  /sadasdasdsd</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <brush brushstyle="BDiagPattern">
          <color alpha="255">
           <red>161</red>
           <green>197</green>
           <blue>255</blue>
          </color>
         </brush>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="freshButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restoreButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>撤销更改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	
	color: rgb(0, 0, 0);
	background-color: rgb(202, 255, 194);
}
</string>
         </property>
         <property name="text">
          <string>下班</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="commitButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myPainterWidget</class>
   <extends>QWidget</extends>
   <header>myPainterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
